project/src/store/root-reducer.ts

```diff
@@ -0,0 +1,16 @@
+import {combineReducers} from 'redux';
+import {filmData} from './film-data/film-data';
+import {userProcess} from './user-process/user-process';
+
+enum NameSpace {
+  data = 'DATA',
```

Comment:
```suggestion
  Data = 'DATA',
```
ключи с большой буквы